src/utils/Ponyfills.js

```diff
@@ -0,0 +1,15 @@
+// This file is intented to have multiple exports
+// eslint-disable-next-line import/prefer-default-export
+export function convertUint8ArrayToString(array) {
+    if (window.TextDecoder !== undefined) {
+        return new TextDecoder().decode(array);
+    }
+
+    // Avoid the String.fromCharCode.apply(null, array) shortcut, which
+    // throws a "maximum call stack size exceeded" error for large arrays.
+    var s = '';
+    for (var i = 0, il = array.length; i < il; i++) {
+        s += String.fromCharCode(array[i]);
```

Comment:
Yep, Unicode code points, so 160 is read as a U+00A0 (no-break space), not as an invalid UTF-8 byte (`10100000` denotes a continuation byte, so it's invalid unless preceeded by a `11xxxxxx` byte) which `TextDecoder` would replace with a U+FFFD (replacement character).